docs/Migrate-from-V1-to-V2.md

```diff
@@ -107,7 +107,7 @@ As a result, your Survey Creator component code can look as follows:
 ```javascript
```

Comment:
Can you make the following changes to the above lines?

Line 11:
Can you update this line as follows?

`The biggest change is native support for the [React](https://reactjs.org/) JavaScript framework. We also plan to add similar native support for other JavaScript frameworks such as [Angular](https://angular.io/) and [Vue](https://vuejs.org/). For this purpose, we had to reorganize our source code and divide it into framework-related packages. As a result, we divided our Knockout.js code base into parts to emphasize a common core part and detach parts for individual frameworks.`

Line 16:
Can you update this line as follows?

`We changed the SurveyJS Creator's UI dramatically. The new UI is not merely a face lift. We redesigned the UI based on the constructive and reasonable feedback from our customers and their users.`

Line 18:
`Remarkable changes have been made to the following [UI elements](https://surveyjs.io/Documentation/Survey-Creator?id=End-User-Guide#basics):`

Please change "Remarkable" to "Significant".

Lines 20-23:

Please remove the punctuation (commas and a period) from these list items.

Line 28:
Can you update this line as follows?

`And since the SurveyJS Library already supports different frameworks and has been thoroughly tested for accessibility and important functional requirements and use-cases, it empowers us to support the React framework in V2 (and to implement support for other platforms in the same way in the near future). Although this approach exposes some incompatibilities with V1 (for example, in creating adorners and custom property editors), we strongly believe that the benefits outweigh the costs.`

Line 30:
Can you update this line as well?

`To facilitate your migration from V1 to V2, we log potential incompatibilities as warnings into the Console when possible. The warning specifies the function or property that does not work in V2 and suggests that you use another technique or different parameters.`

Line 43:
What do you think about the following alternative? Note that I changed the SurveyJS link to the actual topic title.
Let me know what you think.

`We still use the same base library (SurveyJS Library). However, for V2 of Creator you need to refer to the SurveyJS Library as a library divided into two modules: a framework-independent part (core) and a framework-dependent part (Knockout in our case). Find and replace the following links (as mentioned in the [Get Started with Survey Creator / Form Builder](https://surveyjs.io/Documentation/Survey-Creator?id=Add-Survey-Creator-into-your-Web-Page#step-2.add-links-to-survey-creator-and-survey-library) topic):`

Line 88:
`If you pass a DOM element or an element id into a SurveyCreator constructor as the first parameter, we (in V2) log a warning in the Console and ignore this constructor parameter in code.`

Can you make "we" more specific?
For example, do you mean "...SurveyCreator (in V2) logs a warning in the Console and ignores this constructor parameter in code."

Line 96:
Can you change "provide" to "include"?

Line 98:
What do you mean by "will go into trash"? For example, do you mean "will be deleted"?

Line 99:
When you say "rendering", do you mean that _content_ is rendered?
If so, can we say the following instead?

`The majority of the code is still valid, except for the code that renders content and implements other advanced functionality, such as custom property editors.`

Line 101:
Please add an article ("the") before "correct packages".

Line 102:
Is `survey-creator-react` a form builder package? If so, can we say something like the following?

`You need to install `survey-creator-react`. This form builder depends on `survey-creator-core`, `survey-react-ui`, and `survey-core`, which will be installed as well.`